modules/carto/src/api/maps-v3-client.js

```diff
@@ -307,7 +308,7 @@ export async function fetchMap({cartoMapId, clientId, credentials, autoRefresh,
   }
 
   const url = `${localCreds.mapsUrl}/public/${cartoMapId}`;
-  const map = await request({url});
+  const map = await request({url, accessToken});
```

Comment:
@eamador token is received, but it is not being included in the fetch headers, right?